WowPacketParserModule.V9_0_1_36216/Parsers/GuildHandler.cs

```diff
@@ -9,13 +9,13 @@ public static class GuildHandler
         [Parser(Opcode.CMSG_GUILD_INVITE_BY_NAME, ClientVersionBuild.V9_1_0_39185)]
         public static void HandleGuildInviteByName(Packet packet)
         {
-            var bits16 = packet.ReadBits(9);
-            var unused910 = packet.ReadBit();
+            var nameLength = packet.ReadBits(9);
+            var isArena = packet.ReadBit("IsArenaTeamInvite");
```

Comment:
Bits like that dont actually have their own name, this is just the "has_value" of Optional<int32> (in WPP you would typically write "HasX" where "X" is the name for that field, in this case HasArenaTeamId)